src/k8s/pkg/k8sd/database/sql/queries/worker-tokens/delete-by-token.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM
+    worker_tokens AS t
+WHERE
+    t.token = ?
```

Comment:
What was the reasoning behind deleting by token and not the id (primary key)?

Reply:
This will delete the database row that matches the token. The code that deletes the token does not know the id.